common/src/main/java/com/wynntils/commands/LootrunCommand.java

```diff
@@ -0,0 +1,404 @@
+            }
+        }
+        return 0;
+    }
+
+    private int addJsonLootrunNote(CommandContext<CommandSourceStack> context) {
+        Component text = ComponentArgument.getComponent(context, "text");
+        Entity root = McUtils.player().getRootVehicle();
+        BlockPos pos = root.blockPosition();
+        context.getSource()
+                .sendSuccess(
+                        new TranslatableComponent("feature.wynntils.lootrunUtils.addedNote", pos.toShortString())
+                                .append("\n" + text),
+                        false);
+        return LootrunUtils.addNote(text);
+    }
+
+    private int addTextLootrunNote(CommandContext<CommandSourceStack> context) {
+        Component text = new TextComponent(StringArgumentType.getString(context, "text"));
+        Entity root = McUtils.player().getRootVehicle();
+        BlockPos pos = root.blockPosition();
+        context.getSource()
+                .sendSuccess(
+                        new TranslatableComponent("feature.wynntils.lootrunUtils.addedNote", pos.toShortString())
+                                .append("\n" + text),
+                        false);
+        return LootrunUtils.addNote(text);
+    }
+
+    private int listLootrunNote(CommandContext<CommandSourceStack> context) {
+        LootrunUncompiled current = LootrunUtils.getActiveLootrun();
+
+        if (current == null) return 0;
+
+        if (current.notes().isEmpty()) {
+            context.getSource().sendFailure(new TranslatableComponent("feature.wynntils.lootrunUtils.listNoteNoNote"));
+        } else {
+            MutableComponent component = new TranslatableComponent("feature.wynntils.lootrunUtils.listNoteHeader");
+            for (Pair<Vec3, Component> note : current.notes()) {
+                BlockPos pos = new BlockPos(note.first());
+                String posString = pos.toShortString();
+
+                component
+                        .append("\n")
+                        .append(new TextComponent("[X]").withStyle((style) -> style.withHoverEvent(new HoverEvent(
+                                        HoverEvent.Action.SHOW_TEXT,
+                                        new TranslatableComponent("feature.wynntils.lootrunUtils.listClickToDelete")))
+                                .withClickEvent(new ClickEvent(
+                                        ClickEvent.Action.RUN_COMMAND,
+                                        "/lootrun note delete " + posString.replace(",", "")))
+                                .withColor(ChatFormatting.RED)))
+                        .append(" " + posString + ": ")
+                        .append(note.second());
+            }
+            context.getSource().sendSuccess(component, false);
+        }
+        return 1;
+    }
+
+    private int deleteLootrunNote(CommandContext<CommandSourceStack> context) throws CommandSyntaxException {
+        LootrunUncompiled current = LootrunUtils.getActiveLootrun();
```

Comment:
same here - a `deleteNote` function that returns a boolean for success would be cleaner